yt_dlp/extractor/pialive.py

```diff
@@ -71,11 +72,32 @@ def _real_extract(self, url):
             program_code, data=payload, headers={'Content-Type': content_type, 'Referer': self.PLAYER_ROOT_URL},
         )
 
+        article_code = self._search_regex(r"const articleCode = '(.*?)';", webpage, 'article code')
+        chat_info = self._download_json(
+            f'{self.PIA_LIVE_API_URL}/perf/chat-tag-list/{program_code}/{article_code}',
+            article_code, data=payload, headers={'Content-Type': content_type, 'Referer': self.PLAYER_ROOT_URL},
+        )['data']['chat_one_tag']
+        chat_room_url = extract_attributes(chat_info)['src']
+        comment_page = self._download_webpage(
+            chat_room_url, program_code, headers={'Referer': f'{self.PLAYER_ROOT_URL}/'}, note='Fetching comment page', errnote='Unable to fetch comment page')
```

Comment:
`f'{self.PLAYER_ROOT_URL}/'}` , The trailing slash at the end is to ensure the correct Referer and prevent a 403 error.